packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -1,14 +1,74 @@
 import * as React from 'react';
 import Onboarding from 'Components/onboarding';
 import { trading_hub_contents } from 'Constants/trading-hub-content';
-import AddOptionsAccount from 'Components/add-options-account';
+// import AddOptionsAccount from 'Components/add-options-account';
+import Joyride from 'react-joyride';
+import ToggleAccountType from 'Components/toggle-account-type';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
 
 const TradingHub = () => {
     /*TODO: We need to show this component whenever user click on tour guide button*/
+    const [is_tour_open, setIsTourOpen] = React.useState(false);
+
     return (
         <React.Fragment>
-            <AddOptionsAccount />
-            <Onboarding contents={trading_hub_contents} />
+            {/* <AddOptionsAccount /> */}
+            <ToggleAccountType accountTypeChange={undefined} value={''} />
```

Comment:
```suggestion
            <ToggleAccountType value={''} />
```
Doing this will also cause `accountTypeChange` to be undefined.